modules/create_book.rb

```diff
@@ -0,0 +1,11 @@
+module CreateBook
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
```

Comment:
- [ ] The "Single-responsibility Principle" state that a class/method should have only one job. Therefore let's make sure that all requests for inputs should be placed in another method/class whose responsibility is to request inputs from the user.

Reply:
@FrederickMih  According to the "single-responsability Principle" this module has one responsibility: to create a book. please kindly check other already accepted projects, who have their setup exactly as we do.

Reply:
Hi @jfoyarzo, when creating a book, you shouldn't request inputs from the user. Please check out how you list your books and compare with this method.

Reply:
@FrederickMih are you even familiar with this project...? We are not requesting input, this is literally how we can create a book....